apps/dashboard/app/views/files/_files_table.html.erb

```diff
@@ -9,7 +9,9 @@
     <table class="table table-striped table-condensed w-100" id="directory-contents">
       <thead>
         <tr>
-          <th><span class="sr-only">Select</span></th>
+          <th>
+             <input type="checkbox" id="select-all" title="Select All"></input>
```

Comment:
This is a little bit of a nit-pick, but can we use `_` instead of `-` for word separators in ids.

It really is a bit meaningless, but I just feel like we should have uniformity (Rails generates ids with underscores).